compiler/back_end/cpp/header_generator.py

```diff
@@ -374,7 +374,7 @@ def _get_cpp_view_type_for_physical_type(
             element_view_parameter_types="".join(
                 ", " + p for p in element_view_parameter_types),
             element_size=element_size,
-            addressable_unit_size=parent_addressable_unit,
+            addressable_unit_size="1" if parent_addressable_unit == ir_pb2.AddressableUnit.BIT else "8",
```

Comment:
`ir_pb2.AddressableUnit` inherits from int, are the "1" and "8" needed or can this be `str(addressable_unit_size)` or just `addressable_unit_size` directly?

Reply:
Ah okay, so the deal is without this we get the literal string `AddressableUnit.BYTE` in our generated headers instead of `8`. This isn't related to #130. I did find a small backwards compat issue that I'm going to upload where legacy serialization supports the string `BYTE` in json. This isn't an issue later on in my patch series but I'd like the tests to not fail!